packages/arb-node-core/nodehealth/nodehealth.go

```diff
@@ -151,38 +164,24 @@ func checkInboxReader(config *configStruct, state *healthState) healthcheck.Chec
 	check := healthcheck.Async(func() error {
 		//Lock config mutex for read operation
 		state.mu.Lock()
+		defer state.mu.Unlock()
 		//Calculate out the block difference
 		const subtractionBigInt = 0
-		blockDifference := big.NewInt(subtractionBigInt).Sub(&state.inboxReader.caughtUpTarget, &state.inboxReader.currentHeight)
+		blockDifference := big.NewInt(subtractionBigInt).Sub(state.inboxReader.caughtUpTarget, state.inboxReader.currentHeight)
 		//Set the tolerance we are willing to accept
 		tolerance := big.NewInt(config.blockDifferenceTolerance)
 		//Compare the tolerance using CmpAbs, fail if > then tolerance
 		const greaterThanReturn = 1
 		if blockDifference.CmpAbs(tolerance) == greaterThanReturn {
```

Comment:
Our standard way of doing this is just directly `> 0` instead of using a var and `== greaterThanReturn`